fem/libceed/mass.cpp

```diff
@@ -23,6 +23,10 @@ void CeedPAMassAssemble(const FiniteElementSpace &fes,
                         const mfem::IntegrationRule &irm, CeedData& ceedData)
 {
 #ifdef MFEM_USE_CEED
+   Mesh &mesh = *fes.GetMesh();
+   // Perform checks for some assumptions made in the Q-functions.
+   MFEM_VERIFY(mesh.Dimension() == mesh.SpaceDimension(), "case not supported");
```

Comment:
If you know the formula for the other cases, I can implement them in an other PR.

Reply:
No need to do it right now. This was just to inform/protect users who may try to use this case.

Reply:
For completeness: in the general case, `det(J)` is replaced by `[det(J^T J)]^{1/2}`, where `J` is the `SpaceDimension` x `Dimension` Jacobian matrix.